rest/authorization-server/src/main/java/org/eclipse/sw360/rest/authserver/security/Sw360SecurityEncryptor.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Siemens AG, 2018. Part of the SW360 Portal Project.
+ *
+ * SPDX-License-Identifier: EPL-1.0
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.eclipse.sw360.rest.authserver.security;
+
+import org.apache.log4j.Logger;
+
+import javax.crypto.*;
+import javax.crypto.spec.SecretKeySpec;
+import java.io.IOException;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+
+public class Sw360SecurityEncryptor {
+
+    private static final Logger log = Logger.getLogger(Sw360SecurityEncryptor.class);
+
+    private static final byte[] SECRET_SPEC_KEY = "sw360-rest-super-secret-password".getBytes();
+    private static final String SECRET_SPEC_TRANSFORMATION = "AES";
+
+    public static SealedObject encrypt(String password) throws IOException {
+        if (password != null && !password.isEmpty()) {
```

Comment:
Why don't you support the empty password? Looks like an unnecessary restriction to me.

Reply:
Yes it should support also a empty password.
Removed `!password.isEmpty()` condition.